### PR TITLE
JAMES-1914 Add integration test between IMAP and JMAP

### PR DESCRIPTION
**Pull request by aduprat**
Only next commit to be review (the other commits were already reviewed here: #587)

### CODE REVIEW

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/ImapSetMessagesMailboxesUpdatesCompatibility.feature [hunk 1]
```diff
@@ -30,8 +30,17 @@ Feature: IMAP compatibility of JMAP setMessages method used to update mailboxes
     Given the user has a message "m1" in "source" mailbox with subject "My awesome subject", content "This is the content"
     Given the user has an open IMAP connection with mailbox "mailbox" selected
     When the user move "m1" to mailbox "mailbox"
+    Then mailbox "mailbox" contains 1 messages
     Then the user has a IMAP RECENT and a notification about 1 new messages on connection for mailbox "mailbox"
 
+  Scenario: If a message is copie by IMAP, JMAP should see the message and IMAP client that have selected the destination mailbox will be notified
```
Comment (rouazana) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/ImapSetMessagesMailboxesUpdatesCompatibility.feature]: should be a MPT test, no?

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/ImapSetMessagesMailboxesUpdatesCompatibility.feature [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Then the user has a IMAP notification about 1 new message when selecting mailbox "mailbox"
+
+  Scenario: If a message is moved by JMAP, IMAP client that have selected the destination mailbox will be notified
+    Given the user has a message "m1" in "source" mailbox with subject "My awesome subject", content "This is the content"
+    Given the user has an open IMAP connection with mailbox "mailbox" selected
+    When the user move "m1" to mailbox "mailbox"
+    Then mailbox "mailbox" contains 1 messages
+    Then the user has a IMAP RECENT and a notification about 1 new messages on connection for mailbox "mailbox"
+
+  Scenario: If a message is copie by IMAP, JMAP should see the message and IMAP client that have selected the destination mailbox will be notified
```
Comment (rouazana) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/ImapSetMessagesMailboxesUpdatesCompatibility.feature]: *copied

**Comment by aduprat** [comment]
test this please

**Pull request merged**